### ISSUE TITLE
bug: report scheme error when event record

### ISSUE THREAD

**Issue by gxthrj** [apache/apisix-ingress-controller#468]
### Issue description
Report scheme error when event record: no kind is registered for the type v2alpha1.ApisixRoute in scheme "pkg/runtime/scheme.go:100"
![image](https://user-images.githubusercontent.com/4413028/118761807-ab6c9000-b8a7-11eb-9bdf-5f3d47db5550.png)

### Environment

* your apisix-ingress-controller version : master;

**Comment by LuChenjing** [comment]
Met the same issue with above screentshot.
apisix version: apisix-2.1-2.noarch
apisix ingress-controller: 0.6.0


apisix ingress config likes:
```yaml
apiVersion: apisix.apache.org/v2alpha1
kind: ApisixRoute
metadata:
  name: grafana-route
  namespace: onest-observer
spec:
  http:
  - name: grafana
    match:
      paths:
      - "/grafana*"
    backend:
     serviceName: grafana-ha-svc
     servicePort: 3000
```

**Comment by tao12345666333** [comment]
@gxthrj  Please set milestone for this issue

/milestone v1.1

**Comment by tokers** [comment]
> @gxthrj  Please set milestone for this issue
> 
> 
> 
> /milestone v1.1

Done.

**Comment by gxthrj** [comment]
This has been fixed in #469 , let`s close this.
